portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,22 @@ function addRandomQuote() {
   quoteContainer.innerText = quote;
 }
 
+async function showHelloWorld() {
+  const responseFromServer = await fetch('/hello');
+  const textFromResponse = await responseFromServer.text();
+ 
+  const dateContainer = document.getElementById('hello-container');
```

Comment:
Does this work? Maybe I'm just totally missing it, but I don't see this hello-container element in your html.